docs/Gallery.md

```diff
@@ -30,23 +30,25 @@ Instructions for adding images or videos:
 
 This page highlights some visualizations of SpECTRE simulations.
 
-## Adaptive mesh refinement
+\note We're always happy to feature images and videos created by the SpECTRE
+community on this page. See [this page on
+GitHub](https://github.com/sxs-collaboration/spectre/blob/develop/docs/Gallery.md)
```

Comment:
Isn't this the same page they are currently viewing?

Reply:
Yes but on GitHub, where the comments at the top of the file show up